src/modelinstance.cpp

```diff
@@ -1397,6 +1428,26 @@ Status ModelInstance::infer(const RequestType* requestProto,
     // handleCallback(requestProto, responseProto); to be enabled when callbacks are implemented in network API's
     return status;
 }
+void ModelInstance::checkForOutputTensorResetAbility() {
+    StreamIdGuard guard(getInferRequestsQueue());
+    auto request = guard.getInferRequest();
+    bool allOutputsSupported = true;
+    for (auto [name, _] : getOutputsInfo()) {
+        try {
+            ov::Tensor tensor = request.get_tensor(name);
```

Comment:
Is it possible that we capture input name here? If 	model has an input called `name`?

Reply:
Output name is logged in L1442/1439